gui/GameConqueror.py

```diff
@@ -1043,8 +1043,8 @@ def check_backend_version(self):
 if __name__ == '__main__':
     # Parse cmdline arguments
     parser = argparse.ArgumentParser(prog='GameConqueror',
-                                     description="A GUI for scanmem, a game hacking tool"
-
+                                     description="A GUI for scanmem, a game hacking tool",
+                                     epilog='Report bugs to <%s>.' %(PACKAGE_BUGREPORT,)
```

Comment:
I modelled it on scanmem's help format, you'll want to change it over there, too.